### DISCUSSION TITLE
「贴图」指令为什么是自动截取全屏呢？

### DISCUSSION THREAD

**Discussion by Giresharu** [Snipaste/feedback#2977] [Q&A] [answered]
我所理解的贴图是把截取的图片置顶渲染，如普通截图后按下中键后的效果。
而快捷键中的，以及托盘菜单中的「贴图」指令，却是截取全屏然后置顶渲染。
是我对这个功能的理解不对吗？

**Comment by Geeyun-JY3** [comment]
无法复现，建议提供更多信息如录屏。

**Reply by Giresharu** [reply, marked as answer]
> 无法复现，建议提供更多信息如录屏。

突然意识到贴图是贴剪贴板里的图，这下没事了🤣
我就说为什么一直给我截全屏，原来是我剪贴板里一直有一张全屏的截图
